### DISCUSSION TITLE
Expand collapse container

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7562] [Ideas]
Add an option to the container that would add a +/- icon to expand or collapse the container.

Useful for example for the attachment field that is very big.

**Comment by melohagan** [comment]
Hey @NicolasFabre81 

I like the suggestion, it would be a handy setting!

There are examples of ways to build this at the moment. For example: 

- https://github.com/Budibase/budibase/discussions/5763
- https://github.com/Budibase/budibase/discussions/3223#discussioncomment-2993156 (took me a while to dig this one out!)
